### DISCUSSION TITLE
Bluetooth Serial Conection and write

### DISCUSSION THREAD

**Discussion by lajith111** [capacitor-community/bluetooth-le#627] [General]
Currently Bluetooth printer also support  dual mode Serial connection with BLE . Currently no plugin supports for serial connection..
Whether this plugin connect / Write  limited feature through serial.? I am using this plugin for BLE.. Thanks for this plugin

**Comment by pwespi** [comment]
This plugin only supports BLE and adding Bluetooth Serial is currently not planned.

**Reply by lajith111** [reply]
Ok.. thanks
